src/plugins/graph/imports/components/axis/components/axis-or-legend-attribute-menu.tsx

```diff
@@ -89,9 +95,20 @@ const _AxisOrLegendAttributeMenu = ({ place, attributeId, target, portal, onOpen
                     Link Data
                   </MenuItem>
                 }
-                { data?.attributes?.map((attr) => {
+                { data?.attributes?.map((attr, idx) => {
+                  console.log("\t🥩 attr.Name:", attr.name);
+                  //only show y attr that is not self, not the x axis, and not an already plotted Y
+                  const isCurrent = attr.id === attributeId;
+                  const isXAxis = (idx === 0); // maybe replace with const isXaxis = (idx === 0 && attr.name === "x");
```

Comment:
Should this also enforce that then name is "x"? 
As in replace this with 
`const isXaxis = (idx === 0 && attr.name === "x");`